tests/e2e/linalg_ext_ops/BUILD.bazel

```diff
@@ -39,6 +42,22 @@ iree_check_single_backend_test_suite(
     target_backend = "llvm-cpu",
 )
 
+iree_check_single_backend_test_suite(
+    name = "check_llvm-cpu_local-task",
+    srcs = [
+        "attention_i1_mask.mlir",
+    ],
+    compiler_flags = [
+        "--iree-llvmcpu-target-cpu=generic",
+        "--iree-experimental-packed-i1-storage",
+    ],
+    driver = "local-task",
+    tags = [
+        "nowasm",
+    ],
```

Comment:
Is the nowasm tag necessary? Ideally these tests would also work there too.

If the tests are failing, please copy the comment like above:
```
        # attention fails with a wasm target, just disable the tests there for now
        #   error: Yield operand #2 is not equivalent to the corresponding iter bbArg
```

Reply:
Thanks. Missed that. The original tests do not work for wasm so this will not work either.